apps/src/lab2/views/components/Instructions/NavigationArea.tsx

```diff
@@ -74,8 +75,18 @@ const NavigationArea: React.FC<NavigationAreaProps> = ({
   const validationSatisfied = useAppSelector(
     state => state.lab.validationState?.satisfied
   );
+  // Get the level number of the next level (in the progression)
+  const continueToLevel = useAppSelector(
```

Comment:
Nit: could name something like `nextLevelNumber` and `currentLevel` as `currentLevelNumber` (below).